From: Reinhard Tartler <siretart@tauware.de>
Date: Wed, 10 Jul 2024 21:16:08 -0400
Subject: Disable TestDeviceFromPath

Closes: #1072147
---
 pkg/parse/parse_test.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pkg/parse/parse_test.go b/pkg/parse/parse_test.go
index f5b12e1..61aec70 100644
--- a/pkg/parse/parse_test.go
+++ b/pkg/parse/parse_test.go
@@ -88,6 +88,7 @@ func TestIsValidDeviceMode(t *testing.T) {
 }
 
 func TestDeviceFromPath(t *testing.T) {
+	t.Skip("Test unsuitable for debian, see https://bugs.debian.org/1072147")
 	if runtime.GOOS != "linux" {
 		t.Skip("Devices is only supported on Linux")
 	}
